libs/sdk-ui-kit/src/SettingItem/typings.ts

```diff
@@ -0,0 +1,36 @@
+// (C) 2022 GoodData Corporation
+
+import { ReactElement } from "react";
+
+/**
+ * @internal
+ */
+export enum SETTING_ITEM_ACTION_TYPE {
```

Comment:
Please rename it `SETTING_ITEM_ACTION_TYPE` → `SettingItemActionType` to conform with the naming convention for other enums, such as:

- https://github.com/gooddata/gooddata-ui-sdk/blob/b6f6080757af49db8eeb8d71dea39eee3ddab4e9/libs/sdk-embedding/src/iframe/common.ts#L11
- https://github.com/gooddata/gooddata-ui-sdk/blob/b6f6080757af49db8eeb8d71dea39eee3ddab4e9/libs/sdk-embedding/src/iframe/common.ts#L28
- https://github.com/gooddata/gooddata-ui-sdk/blob/b6f6080757af49db8eeb8d71dea39eee3ddab4e9/libs/sdk-embedding/src/iframe/common.ts#L96

Reply:
fixed